i3lock.c

```diff
@@ -632,6 +633,36 @@ void handle_screen_resize(void) {
     redraw_screen();
 }
 
+static bool verify_png_image(const char *image_path) {
+    if (!image_path) {
+        return false;
+    }
+
+    /* Check file exists and has correct PNG header */
+    FILE *png_file = fopen(image_path, "r");
+    if (png_file == NULL) {
+        fprintf(stderr, "Image file path \"%s\" cannot be opened: \"%s\"\n", image_path, strerror(errno));
```

Comment:
No need for the quotes around the error message. The quotes are nice to ensure the file name is clearly separated from the remainder of the message, but strerror’s return value is supposed to blend nicely into error messages. Let’s remove the quotes.